l3kernel/l3bootstrap.dtx

```diff
@@ -69,12 +69,12 @@
 %   \begin{syntax}
 %     \cs{ExplSyntaxOn} \meta{code} \cs{ExplSyntaxOff}
 %   \end{syntax}
-%   The \cs{ExplSyntaxOn} function switches to a category code
-%   regime in which spaces and new lines are ignored, and in which the colon (|:|)
+%   The \cs{ExplSyntaxOn} function modifies the current category code
+%   regime such that spaces and new lines are ignored, the colon (|:|)
 %   and underscore (|_|) are treated as \enquote{letters}, thus allowing
 %   access to the names of code functions and variables. Within this
 %   environment, |~| is used to input a space. The \cs{ExplSyntaxOff}
-%   reverts to the document category code regime.
+%   reverts to their standard values what \cs{ExplSyntaxOn} has modified.
```

Comment:
This sentence doesn't make sense.

Reply:
Is it more clear?
```
The \cs{ExplSyntaxOff} reverts what \cs{ExplSyntaxOn} has modified to their standard values.
```

Reply:
I thing something like `reverts category codes modified by \cs{ExplSyntaxOn} to their previous values.` might be better.

Reply:
yes as the point is it doesn't reset to a pre-defined default it just undoes the previous ExplSyntaxOn. Also we should probably mention that `\endlinechar` as well as catcodes are (re)set.

Reply:
not only the category codes but also the value of `\endlinechar`.
A different option is to detail exactly what is changed ?